include/soci/empty/soci-empty.h

```diff
@@ -135,20 +135,20 @@ struct empty_rowid_backend : details::rowid_backend
 {
     empty_rowid_backend(empty_session_backend &session);
 
-    ~empty_rowid_backend();
+    ~empty_rowid_backend() SOCI_OVERRIDE;
```

Comment:
FWIW I'm personally for using `override` on dtors too but this is recommended _against_ by C++ core guidelines, see [this closed issue](https://github.com/isocpp/CppCoreGuidelines/issues/721).

Reply:
Interesting reading. I've started using the CppCoreGuidelines, but not relying on it strictly due to the performance issues (eg. with `gsl::span`).

Meanwhile, I agree with your comment in that thread.

For now, I'll stick `SOCI_OVERRIDE` in destructors. After we move to C++11 only and we ever start using the CppCoreGuidelines or discuss/agree to do otherwise. we can always remove it.